### PR TITLE
Add see also section to posts footer

### LINKED ISSUE #150: Expand Series Functionality
I would like to work on this feature but I have some questions.

Since `series` is an array, posts can be in multiple series.

1. How should we go about listing related posts with it?
    - Use first series only?
    - Use all with the first series taking priority?
2. How many related posts should be listed under **See Also** section?
3. Should we make the number of related posts configurable?

## Todo

- [x] Create a See also section at the end of posts and pages.
- [x] Check if the series list pages are ok. And make changes if necessary.
- [x] Translate series to Portuguese. (Hint: It is séries).

_Originally posted by @luizdepra in https://github.com/luizdepra/hugo-coder/pull/147#issuecomment-469277019_

### PR DESCRIPTION
**Pull request by josephting**
### Prerequisites

Put an `x` into the box(es) that apply:

- [ ] This pull request fixes a bug.
- [x] This pull request adds a feature.
- [ ] This pull request introduces breaking change.

### Description

Posts that has `series` in its front matter will get listed at the footer of the post. Links to related posts of the same series will be listed beneath.

5 (configurable with `seriesRelatedCount`) related posts will be displayed by default. Had to add 1 in order to include the current post. Very bad practice but my hugo & golang knowledge is limited. Any help or advise are appreciated.

### Issues Resolved

Resolves #150.

### Checklist

Put an `x` into the box(es) that apply:

#### General

- [x] Describe what changes are being made
- [x] Explain why and how the changes were necessary and implemented respectively
- [x] Reference issue with `#<ISSUE_NO>` if applicable

#### Resources

- [x] If you have changed any SCSS code, run `make release` to regenerate all CSS files

#### Contributors

- [x] Add yourself to `CONTRIBUTORS.md` if you aren't on it already

### CODE REVIEW

File: layouts/partials/posts/series.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{{ $currentPageUrl := .URL }}
+{{ if .Params.series }}
+<section class="see-also">
+  {{ range .Params.series }}
+    <h3>See also in {{ . }}</h3>
+    {{ $name := . | urlize }}
+    {{ $series := index $.Site.Taxonomies.series $name }}
+    {{ if gt (len $series.Pages) 0 }}
+      <nav class="list">
```
Comment (luizdepra) [inline layouts/partials/posts/series.html]:
```suggestion
      <nav>
```
No need to use this class here.

File: layouts/partials/posts/series.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        <ul>
+        {{ range first (add $.Site.Params.seriesRelatedCount 1 | default 6) $series.Pages }}
+          {{ if ne .URL $currentPageUrl }}
+            <li>
+              <a class="title" href="{{ .Params.ExternalLink | default .URL }}">{{ .Title }}</a>
```
Comment (luizdepra) [inline layouts/partials/posts/series.html]:
```suggestion
              <a href="{{ .Params.ExternalLink | default .URL }}">{{ .Title }}</a>
```
Remove this class too. The list looks nicier with the link look.

File: assets/scss/_content.scss [hunk 3]
```diff
@@ -19,6 +19,16 @@
     }
     footer {
       margin-top: 4.0rem;
+      .see-also {
+        margin: 3.2rem 0;
+        h3 {
+          margin: 3.2rem 0;
+        }
+        ul {
```
Comment (luizdepra) [inline assets/scss/_content.scss]: Without the extra classes, we can remove the `ul` rule here.

File: assets/scss/_content_rtl.scss [hunk 4]
```diff
@@ -15,6 +15,9 @@ body.rtl {
         }
       }
     }
+    article footer .see-also ul {
```
Comment (luizdepra) [inline assets/scss/_content_rtl.scss]: We should revert this change too.

File: exampleSite/config.toml [hunk 5]
```diff
@@ -36,6 +36,9 @@ disqusShortname = "yourdiscussshortname"
     # Use inverted colors
     inverted = false
 
+    # Series see also post count
+    seriesRelatedCount = 5
```
Comment (luizdepra) [inline exampleSite/config.toml]: How about `maxSeeAlsoItems`?

File: layouts/partials/posts/series.html [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+{{ if .Params.series }}
+<section class="see-also">
+  {{ range .Params.series }}
+    <h3>See also in {{ . }}</h3>
+    {{ $name := . | urlize }}
+    {{ $series := index $.Site.Taxonomies.series $name }}
+    {{ if gt (len $series.Pages) 0 }}
+      <nav class="list">
+        <ul>
+        {{ range first (add $.Site.Params.seriesRelatedCount 1 | default 6) $series.Pages }}
```
Comment (luizdepra) [inline layouts/partials/posts/series.html]: Just a minor fix here. Making things more clear.
```suggestion
        {{ $maxItems := $.Site.Params.seriesRelatedCount | default 5 }}
        {{ range first (add $maxItems 1) $series.Pages }}
```

File: assets/scss/_content_rtl.scss (hunk 4 above)
Comment (josephting) [reply assets/scss/_content_rtl.scss]: Completely remove this rule, right?

File: exampleSite/config.toml (hunk 5 above)
Comment (josephting) [reply exampleSite/config.toml]: Thanks for the input! Much better. 💯

File: assets/scss/_content_rtl.scss (hunk 4 above)
Comment (luizdepra) [reply assets/scss/_content_rtl.scss]: yup

**Comment by josephting** [comment]
> I just want to make some minor modifications here.

@luizdepra Thanks for the review. Let me know if there's any other modifications needed.
The `ul` part css I didn't understand. Not sure how removing the class will fix it. 🤔
